core/src/components/toggle/toggle.tsx

```diff
@@ -115,6 +116,13 @@ export class Toggle implements ComponentInterface {
    */
   @Prop() justify: 'start' | 'end' | 'space-between' = 'space-between';
 
+  /**
+   * How to pack the label and control along the cross axis.
+   * `"start"`: The label and control will appear at the top of the container.
+   * `"center"`: The label and control will appear at the center of the container.
+   */
+  @Prop() align: 'start' | 'center' = 'center';
```

Comment:
We're going to need to name this something other than `align` because it collides with the global deprecated `align` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes. The `align` attribute is going to cause the text inside of the label container to be centered.

Maybe we can call this `alignment`? Open to other ideas from the team.

Reply:
You'll also notice that long labels are no longer centered within their container. We're currently discussing the best path forward for this on Slack.

Reply:
`alignment` works for me. And based on the Slack thread, no changes will be needed for centering.